routes.js

```diff
@@ -0,0 +1,12 @@
+async function routes(fastify, options) {
+	fastify.get("/health", async (request, reply) => {
+		return reply.code(200).send({ status: "ok" });
+	});
```

Comment:
if there is a single line function you can 
```suggestion
	fastify.get("/health", async (request, reply) => 
		reply.code(200).send({ status: "ok" });
		)
```